examples/user_guide/Overview.ipynb

```diff
@@ -172,6 +172,7 @@
     "> - `loading_color`: The color of the global loading indicator as a hex color, e.g. #6a6a6a\n",
     "> - `nthreads`: If set will start a `ThreadPoolExecutor` to dispatch events to for concurrent execution on separate cores. By default no thread pool is launched, while setting nthreads=0 launches `min(32, os.cpu_count() + 4)` threads.\n",
     "> - `raw_css`: List of raw CSS strings to add to load.\n",
+    "> - `reuse_sessions`: Whether to reuse a session for the initial request to speed up the initial page render. Note that if the initial page differs between sessions, e.g. because it uses query parameters to modify the rendered content, then this option will result in the wrong content being rendered.\n",
```

Comment:
> Note that if the initial page differs between sessions, e.g. because it uses query parameters to modify the rendered content, then this option will result in the wrong content being rendered.

According to the docs it seems like it's more subtle than that. Maybe linking to the docs section introducing reusable sessions would be better than trying to explain the limitations here?